app/src/main/java/com/example/androidthings/assistant/AssistantActivity.java

```diff
@@ -173,12 +180,23 @@ public void onSpeechRecognition(List<SpeechRecognitionResult> results) {
                     }
                 })
                 .setConversationCallback(new ConversationCallback() {
+                    @Override
+                    public void onResponseStarted() {
```

Comment:
nice catch! I think this was introduce with https://github.com/androidthings/sample-googleassistant/commit/004b0f8e2920b120d9a0eb2d7d3e33e85e94f1dd